src/browser.js

```diff
@@ -10,10 +10,68 @@ exports.formatArgs = formatArgs;
 exports.save = save;
 exports.load = load;
 exports.useColors = useColors;
-exports.storage = 'undefined' != typeof chrome
-               && 'undefined' != typeof chrome.storage
-                  ? chrome.storage.local
-                  : localstorage();
+
+var isFirefox = false;
+var isFirefoxNewerThan31 = false;
+var isAppleWebkit = false;
+
+if (typeof navigator !== 'undefined' &&
+    typeof navigator.userAgent !== 'undefined') {
+  isFirefox = navigator.userAgent.toLowerCase().match(/firefox\/(\d+)/)
```

Comment:
Try my `patch-1-without-ua-sniff` branch https://github.com/andreineculau/debug/tree/patch-1-without-ua-sniff although I find it utterly wrong for an extension to do that even for other extensions.